killrvideo-service-comments/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM openjdk:11
+
+MAINTAINER Cedrick Lunven <cedrick.lunven@datastax.com>
+MAINTAINER Davig Gilardi  <david.gilardi@datastax.com>
```

Comment:
Oops...David, unless you prefer me as Davig

Reply:
Yes, and also I will integrate updates from Aleks